scripts/stac/imagery/item.py

```diff
@@ -46,3 +51,20 @@ def add_collection(self, collection_id: str) -> None:
 
     def add_link(self, rel: str, href: str = "./collection.json", file_type: str = "application/json") -> None:
         self.stac["links"].append({"rel": rel, "href": href, "type": file_type})
+
+    def add_stac_extension(self, link: str) -> None:
+        """add a stac extension link in the 'stac_extensions' list
+
+        Args:
+            link: to the extension json schema
+        """
+        self.stac["stac_extensions"].append(link)
```

Comment:
could be good to check if the extension was already in the list before appending it again.